xml/physical_security.xml

```diff
@@ -426,4 +426,180 @@ polkit.addRule(function(action, subject) {
      </step>
     </procedure>
    </sect1>
+  <sect1 xml:id="sec-USBGuard">
+<title>System protection with <emphasis>USBGuard against rogue USB devices</emphasis></title>
```

Comment:
Personally I find 'rogue USB devices' sounds to much like marketing and not really technical.. Maybe using 'USB security attacks' is another option [0] or something similar to 'intrusive USB devices' [1]. 

The man page [2] talks about 'enforces the USB device authorization', maybe a third option would be to use 'System protection by enforcing USB device authorization'.

What ever works in the big picture, I would advise against using 'rogue'.
[0] https://www.redhat.com/en/blog/built-protection-against-usb-security-attacks-usbguard
[1] https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-using-usbguard
[2] https://man.archlinux.org/man/usbguard-daemon.8.en

Reply:
Sounds good , I will re-phrase